### ISSUE TITLE
[Feature Request] Add an option to disable Dark Read if `color-scheme: dark` exist

### ISSUE THREAD

**Issue by AngeloCore** [darkreader/darkreader#10877]
### Prerequisites

- [X] I [searched for any existing report](https://github.com/darkreader/darkreader/issues?q=is%3Aissue) about this feature request to avoid opening a duplicate.

### Is this feature request related to a problem?

Yes

### Feature Request Description

You can check if dark color scheme is used and disable dark reader if the website already has dark mode

Related:
https://developer.mozilla.org/en-US/docs/Web/CSS/@media/prefers-color-scheme

**Comment by janopae** [comment]
I think this already is implemented in the "new design". If you click on "Settings", you'll see a window with a lot of CSS code, and at the bottom, there will be a button called "Switch to new design". This button will enable a lot of useful features, such as not altering websites that already support dark mode. It also adds the option to follow the system dark theme setting.

After you enabled the "new design", you will see a button called "Detect dark theme" under "Settings". This will detect whether the website already provides a dark theme and not apply Darkreader on these pages by default. 

Funnily, I just installed the extension and missed these two features and went here on GitHub to see if there is a feature request and found yours. But playing around with the extension and reading other Issues, I just found the "New design" button which resolves this for me.
